cypress/integration/flows_page_test.js

```diff
@@ -0,0 +1,80 @@
+describe('Flows page tests', () => {
+  context('no access before login', () => {
+    it('redirects to login', () => {
+      cy.visit('/flows');
+      cy.location('pathname').should('eq', '/login');
+    });
+  });
+
+  context('authenticated', () => {
+    beforeEach(() => {
+      cy.fixture('flows').as('flows');
+      cy.fixture('flow.room1-occupation').as('flow');
+      cy.server();
+      cy.route('GET', '/flow/v1/*/flows/*', '@flow').as('getFlow');
+      cy.route({
```

Comment:
you're not using this ROUTE yet, i'd add the test that expects the API call, or remove the route
You can have a look at https://github.com/astarte-platform/astarte-dashboard/blob/release-1.0/cypress/integration/register_device_page_test.js#L74

Reply:
This DELETE route stub is actually already used in "shows a confirm dialog when deleting a flow", where it tries to delete a flow